redfish_utilities/accounts.py

```diff
@@ -110,7 +110,10 @@ def add_user( context, user_name, password, role ):
             account = context.get( account_member["@odata.id"] )
             if account.dict["UserName"] == "" and not account.dict.get( "Enabled", True ):
                 # Empty slot found; PATCH it
-                response = context.patch( account_member["@odata.id"], body = payload, headers = { "If-Match": account.getheader( "ETag" ) } )
+                headers = {}
+                if account.getheader( "ETag" ):
```

Comment:
We might want to have an override flag to activate this logic. Some folks are concerned this is glossing over a potential issue where multiple users could be modifying the same account and potentially locking out the account.

Will be discussing this further with others.

Reply:
From further discussion, we don't think this should be handled automatically like this without a user specifically asking for this. We'd like to see changes so that if a user really wants to bypass required ETag protection for user accounts, they need to call that out as an option from the tool. Would you be able to make changes for this?